integration-tests/test_load.py

```diff
@@ -1,14 +0,0 @@
-import timeit
```

Comment:
Why are you deleting this?

Reply:
Accidentally deleted it. Adding it back.

Reply:
Also, In current state this fails on my local. Does this work for you while running UT?

Reply:
This doesn't need to run, it's just a tool I use for scale testing. I can fix it in a different PR